activity_browser/ui/tables/plugins.py

```diff
@@ -36,23 +37,24 @@ def mousePressEvent(self, e):
             proxy = self.indexAt(e.pos())
             if proxy.column() == 0:
                 new_value = not bool(proxy.data())  
-                plugin_name = self.model.get_plugin_name(proxy)
-                if new_value:
-                    signals.plugin_selected.emit(plugin_name)
-                else:
+                plugin_name = self.model.get_plugin_name(proxy) #
+                if not new_value:
                     msgBox = QMessageBox()
                     msgBox.setText("Remove plugin from project ?")
                     msgBox.setInformativeText("This will removed all data created by the plugin.")
```

Comment:
This will _remove_ all data created by the plugin

Reply:
Who would have thought I'm a native speaker!